force-app/main/default/lwc/paycheckBreakdown/paycheckBreakdown.js

```diff
@@ -0,0 +1,78 @@
+import { LightningElement, track, wire } from 'lwc';
+import { getRecord, getFieldValue } from 'lightning/uiRecordApi';
+import getCalculateNetTakeHomePay from '@salesforce/apex/TaxCalculationHandler.getCalculateNetTakeHomePay';
+import getPaymentIntervals from '@salesforce/apex/TaxCalculationHandler.getPaymentIntervals'; 
+
+export default class PaycheckBreakdown extends LightningElement {
+    // Initialize to 0
+    @track salary = 0;
+    @track taxRate = 0;
+    @track netPayAnnual = 0; 
+    @track netPaySemiAnnual = 0; 
+    @track netPayMonthly = 0; 
+    @track netPayBiWeekly = 0; 
+
+    @track paymentOptions = []; // For storing payment interval options
+@wire(getPaymentIntervals)
+paymentIntervals({ error, data }) {
+    if (data) {
+        this.paymentOptions = data.map(item => ({
+            label: item.label,
+            value: item.value
+        }));
+    } else if (error) {
+        console.error('Error fetching payment intervals:', error);
+        this.paymentOptions = [];  
+    }
+}
+
+    // Handle changes in salary input and reinitialize pay breakdowns
+    handleSalaryChange(event) {
+        this.salary = parseFloat(event.target.value); // Parse input as float to ensure correct data type
+        this.resetPayBreakdown(); // Reset pay breakdown values to 0
+    }
+
+    // Reset all computed pay breakdowns to zero
+    resetPayBreakdown() {
+        this.netPayAnnual = 0;
+        this.netPaySemiAnnual = 0;
+        this.netPayMonthly = 0;
+        this.netPayBiWeekly = 0;
+    }
+
+handleCalculate() {
+    getCalculateNetTakeHomePay({ salary: this.salary })
+        .then(result => {
+            if (result) {
+                this.netPayAnnual = result.Take_Home_Pay__c;
```

Comment:
These calculations are being done in `TaxCalculationHandler.calculateTaxesAndTakeHomePay()` and saved to the job application, so there might be a way to break out that logic and avoid duplicating your efforts